### DISCUSSION TITLE
can't switch to admin from personal

### DISCUSSION THREAD

**Discussion by bmlimone** [community/community#64532] [Enterprise]
I am trying to switch from my personal account to my admin account. I am being billed for 8 seats and I can't downgrade. When I look in my personal profile it states my billing is $0 and I have no organizations and enterprises.  I need to find where I can stop this billing! Thank you!

**Comment by arafajs** [comment]
- Check your billing settings. Make sure that you are not subscribed to any paid plans.
- Check your organization memberships. Make sure that you are not a member of any organizations that are being billed.
- Contact the organization owner. If you are still a member of an organization that is being billed, contact the organization owner and ask them to remove you from the organization.
- Otherwise, Go to the GitHub Support website: https://support.github.com/. Click on the "Contact support" button. Select the "Billing" topic. Fill out the form and explain your situation.

**Reply by bmlimone** [reply]
Thank you. I was the admin now I cannot access it. There are 0 members and 0 repositories, and I am still being billed for 8 seats. I am the original admin for the account. I will contact support.
